src/main/java/seedu/duke/command/FindCommand.java

```diff
@@ -22,6 +29,11 @@ public FindCommand(String keywords) {
 
     @Override
     public String execute() {
+        ArrayList<Note> filteredNotes = (ArrayList<Note>) notebook.getNotes().stream()
+                .filter((s) -> s.getTitle().contains(keywords))
+                .collect(Collectors.toList());
+
+        printFindNotesMessage(filteredNotes);
         return null;
```

Comment:
I am thinking that the printFindNotesMessage in InterfaceManager can be a static function and it returns a string of the formatted note. This is to consolidate all the printing to be only done inside the main runLoop function

Reply:
Will not print, will return a String instead